src/bun.js/api/bun/subprocess/ResourceUsage.zig

```diff
@@ -63,11 +63,10 @@ pub fn getContextSwitches(this: *ResourceUsage, globalObject: *JSGlobalObject) J
 }
 
 pub fn finalize(this: *ResourceUsage) callconv(.C) void {
-    bun.default_allocator.destroy(this);
+    bun.destroy(this);
 }
 
 const bun = @import("bun");
-const default_allocator = bun.default_allocator;
 const Rusage = bun.spawn.Rusage;
 
 const jsc = bun.jsc;
```

Comment:
was this a related change?

Reply:
Yeah, was causing ASAN issues. happened in the recent subprocess splitup and only happens on macos. surprised it wasnt ran into already in other places